Client/Frontend/Sync/BraveCore/BraveCoreImportExportUtility.swift

```diff
@@ -190,7 +213,7 @@ extension BraveCoreImportExportUtility {
   func nativeURLPathFromURL(_ url: URL) -> String? {
     return url.withUnsafeFileSystemRepresentation { bytes -> String? in
       guard let bytes = bytes else { return nil }
-      return String(cString: bytes)
+      return String(cString: bytes)//.replacingOccurrences(of: " ", with: "%20").replacingOccurrences(of: "file://", with: "")
```

Comment:
Comment needed?

Reply:
My bad sorry, we were testing something there forgot. 
Fix at https://github.com/brave/brave-ios/pull/6781/commits/e20fbcb4cc732f3fa86e81c4539d698fbe054c60